package.json

```diff
@@ -35,19 +42,27 @@
     "@types/react-resize-detector": "^4.2.0",
     "@types/react-virtualized": "^9.21.2",
     "@types/seedrandom": "^2.4.28",
-    "@types/storybook__addon-knobs": "^5.0.3",
-    "@types/storybook__react": "^4.0.2",
     "@types/uuid": "^3.4.5",
+    "@typescript-eslint/eslint-plugin": "2.x",
+    "@typescript-eslint/parser": "2.x",
     "awesome-typescript-loader": "^5.2.1",
+    "babel-eslint": "10.x",
     "babel-loader": "^8.0.6",
     "carbon-components": "^10.6.0",
     "carbon-components-react": "^7.7.0",
     "carbon-icons": "^7.0.7",
     "cross-env": "^5.1.4",
     "cypress": "3.8.2",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.2.0",
-    "eslint-plugin-prettier": "^3.1.0",
+    "eslint": "6.x",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-config-react-app": "^5.2.0",
+    "eslint-plugin-cypress": "^2.10.3",
+    "eslint-plugin-flowtype": "4.x",
```

Comment:
why are we using a mix of `2.x` and `^2.10.3`? can't we just pick one format?

Reply:
This was based on the command-line here: https://github.com/facebook/create-react-app/tree/master/packages/eslint-config-react-app#usage-outside-of-create-react-app. I'll change those versions to be consistent with rest of file.